nfsstat/check.py

```diff
@@ -50,6 +50,10 @@ def check(self, instance):
         device = Device(this_device, self.log)
         all_devices.append(device)
 
+        # Disregard the first half of device stats (report 1 of 2)
+        # as that is the moving average
+        all_devices = all_devices[len(all_devices) // 2:]
```

Comment:
I tried to think of a better way to do this but I think this might be the best way.